descriptors/ModuleDescriptor-template.json

```diff
@@ -118,6 +118,37 @@
         }
       ]
     },
+    {
+      "id": "invoice-storage.voucher-lines",
```

Comment:
I think it is better to use more general interface name like `voucher-storage` which can cover all endpoints dealing with vouchers

Reply:
agreed. Changed it